core/schema/file.go

```diff
@@ -50,6 +50,25 @@ func (s *fileSchema) Install() {
 			Args(
 				dagql.Arg("name").Doc(`Name to set file to.`),
 			),
+		dagql.NodeFunc("search", DagOpWrapper(s.srv, s.search)).
+			View(AllVersion). // entries returns different results in different versions
+			Doc(`Searches for content matching the given pattern, which may be a regular expression or a literal string.`).
+			Args(
+				dagql.Arg("pattern").Doc(`The text to match.`),
+				dagql.Arg("literal").Doc(`Interpret the pattern as a literal string instead of a regular expression.`),
+				dagql.Arg("multiline").Doc(`Enable searching across multiple lines.`),
+			),
+		dagql.NodeFunc("withReplaced",
+			DagOpFileWrapper(s.srv, s.withReplaced,
+						WithPathFn(keepParentFile[fileReplaceArgs]))).
+			View(AllVersion). // entries returns different results in different versions
```

Comment:
Don't need this here, this is just because `entries` is a weird API that uses the `View` - if we don't use that, we don't need it.